README.md

```diff
@@ -154,6 +154,60 @@ yarn run dev
 ```
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>.
 
+#### Deploying a Custom Image to an OpenShift Cluster
+
+Once you have made changes locally, these instructions will allow you to push changes to an
+ OpenShift cluster for others to review.  This involves building a local image, pushing the
+ image to an image registry, then updating the OpenShift cluster to pull the new image.
+
+##### Prerequisites
+
+1. Docker v17.05 or higher
+2. An image registry like [quay.io](https://quay.io/signin/) or [Docker Hub](https://hub.docker.com/)
+
+##### Steps
+1. Create a repository in the image registry of your choice to hold the image
+2. Build Image `docker build -t <your-image-name> [<your-git-url> | <path-to-dockerfile>]`
+ 
+    Build from Git Repository:
+    ```
+    docker build -t quay.io/myaccount/console:latest github.com/myaccount/console#latestchanges
+    ```
+    ...or from console directory:
+    ```
+    docker build -t quay.io/myaccount/console:latest .
+    ```
+3. Push Image to Image Registry `docker push <your-image-name>`.  Make sure docker is logged into your image registry!  Here we are pushing to a repository named 'console' on quay.io:
+    ```
+    docker push quay.io/myaccount/console:latest
+    ```
+4. Put the console operator in unmanaged state.  Make sure you `oc login` to remote OpenShift cluster. 
```

Comment:
I would leave out the `oc login` sentence. It should be self-evident

Reply:
I stumbled on this step due to not being logged in.  It is self-evident to you experts :-) but not so much to new users.

Reply:
But doesn't `oc` tell you you need to log in?

Reply:
We have so many `oc` commands in the README, and it's not practical to put this note in front of each one. It'd be odd to call it out here and not other places.

Reply:
I stumbled on this step because I wasn't logged into remote cluster.  Again, I don't see the harm in providing this extra info.

Reply:
> But doesn't `oc` tell you you need to log in?

No, I get:
`Unable to connect to the server: dial tcp: lookup api.ui-preserve.origin-gce.dev.openshift.com on 127.0.0.1:53: no such host`
which I think means somehow I'm defaulted to the old team share cluster.
But I get your point.

Reply:
fixed

Reply:
That doesn't mean you're not logged in. It means it can't connect to the cluster (because it's gone).